### PR TITLE
Added Uruguay ID (IC) validator

### LINKED ISSUE #94: [v1.3.0] Implement Uruguay ID 🇺🇾 (CI) validator
- Implement the Uruguay ID validator
- Write tests

### PR DESCRIPTION
**Pull request by gheleri**
resolve #94 
It includes the validator itself and unit tests.

### CODE REVIEW

File: tests/Feature/SouthAmerica/UruguayTest.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            '24895354',
+            '40008026',
+            '60125711',
+        ];
+
+        $this->invalidIds = [
+            '46057422',
+            '90224632',
+            '52631437',
+        ];
```
Comment (AlexOlival) [inline tests/Feature/SouthAmerica/UruguayTest.php:33]: Can we get at least two more invalid ids?

File: src/Core/SouthAmerica/Uruguay/UruguayIdValidator.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        $id = str_pad($id, 7, '0', STR_PAD_LEFT);
+
+        $a = 0;
+
+        $baseNumber = "2987634";
+        for ($i = 0; $i < 7; $i++) {
+            $baseDigit = $baseNumber[$i];
+            $ciDigit = $id[$i];
+
+            $a += (intval($baseDigit) * intval($ciDigit)) % 10;
```
Comment (AlexOlival) [inline src/Core/SouthAmerica/Uruguay/UruguayIdValidator.php:30]: Use a cast to int rather than `intval` for better performance

**Review by AlexOlival — CHANGES_REQUESTED** [review]
Nearly there :+1:

**Comment by gheleri** [comment]
Fixes done

**Pull request merged**